File: Algorithm/Programmers/Python/Level 1/숫자 문자열과 영단어.py
def solution(s):
    list = [
        "zero",
        "one",
        "two",
        "three",
        "four",
        "five",
        "six",
        "seven",
        "eight",
        "nine",
    ]
    for i, letter in enumerate(list):
        s = s.replace(letter, str(i))
    return int(s)

File: Algorithm/Programmers/Java/Level 1/숫자_문자열과_영단어.java
class 숫자_문자열과_영단어 {
    public int solution(String s) {
        String[] dictionary = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };
        for (int i = 0; i < 10; i++) {
            s = s.replace(dictionary[i], i + "");
        }
        return Integer.parseInt(s);
    }
}